fluent-plugin-enhance-k8s-metadata/fluent-plugin-enhance-k8s-metadata.gemspec

```diff
@@ -21,8 +21,9 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency 'concurrent-ruby', '~> 1.1'
   spec.add_runtime_dependency 'fluentd', ['>= 0.14.10', '< 2']
-  spec.add_runtime_dependency 'kubeclient', '4.9.1'
+  # spec.add_runtime_dependency 'kubeclient', '4.9.1' # Git version of Kubeclient specified in Gemfile
```

Comment:
Please create issue to upgrade if new version is available

Reply:
For main repo I mean :)

Reply:
Created: https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/1392